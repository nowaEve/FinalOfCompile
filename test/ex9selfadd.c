int main(){
    int i = 8;
    i++;
    print i;
    i++;
    print i;
    i--;
    print i;
}