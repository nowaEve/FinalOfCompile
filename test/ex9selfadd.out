24 19 0 5 25 13 0 8 12 13 11 13 13 11 0 1 1 12 15 -2 13 11 22 15 -1 13 11 13 13 11 0 1 1 12 15 -2 13 11 22 15 -1 13 11 13 13 11 0 1 2 12 15 -2 13 11 22 21 1